ocl/concepts/tests.py

```diff
@@ -243,6 +243,41 @@ def test_concept_display_name_preferred(self):
         self.assertEquals(self.source1.owner_type, concept.owner_type)
         self.assertEquals(0, concept.num_versions)
 
+    def test_concept_descriptions(self):
+        """
+        Test to make sure that Concept resource is properly updated when adding ConceptDescription to it.
+        """
+        concept = Concept(
+            mnemonic='concept1',
+            created_by=self.user1,
+            updated_by=self.user1,
+            parent=self.source1,
+            concept_class='Diagnosis',
+            names=[self.name],
+            # descriptions=[self.description],
+            datatype="None"
+        )
+
+        description_1 = LocalizedText(
+            name='concept description',
+            locale='en',
+            locale_preferred=True,
+            type='ConceptDescription'
+        )
+
+        concept.descriptions = [description_1]
+        concept.full_clean()
+        concept.save()
```

Comment:
To mimic the issue you should be calling concept_version.clone() and  ConceptVersion.persist_clone(new_version, request.user) same as in https://github.com/OpenConceptLab/oclapi/blob/4d37049cdade81a82178af1b2cbd8509343d1544/ocl/concepts/views.py#L441-L446